public/jobs.js

```diff
@@ -0,0 +1,356 @@
+                    position.value = ''
+                    status.value = 'pending'
+                    thisEvent = new Event('startDisplay')
+                    document.dispatchEvent(thisEvent)
+                } else {
+                    message.textContent = data.msg
+                }
+            } catch(err) {
+                
+                message.textContent = 'A communication error occurred.'
+            }
+        } 
+        suspendInput = false 
+    } 
+    // section 5 -- Edit
+
+    else if (e.target.classList.contains('editButton')) {
+        editJob.dataset.id=e.target.dataset.id
+        suspendInput = true
+        try {
+            response = await fetch(`/api/v1/jobs/${e.target.dataset.id}`, {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Authorization': `Bearer ${token}`
+                }
+            })
+            data = await response.json()
+            if (response.status === 200) {
+                company.value = data.job.company
+                position.value = data.job.position
+                status.value = data.job.status
+                showing.style.display = 'none'
+                showing = editJob
+                showing.style.display = 'block'
+                addingJob.textContent = 'update'
+                message.textContent = ''
+            } else { // might happen if the list has been updated since last display
+                message.textContent = 'The jobs entry was not found'
+                thisEvent = new Event('startDisplay')
+                document.dispatchEvent(thisEvent)
+            }
+        } catch (err) {
+            message.textContent = 'A communications error has occurred.'
+        }
+        suspendInput = false
+    } 
+
+    // section 6 - Delete
+
+    else if (e.target.classList.contains('deleteButton')) {
+        //deleteJob.dataset.id=e.target.dataset.id
+        suspendInput = true
+        try {
+            response = await fetch(`/api/v1/jobs/${e.target.dataset.id}`, {
+                method: 'DELETE',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Authorization': `Bearer ${token}`
+                }
+            })
+            data = await response.json()
+
+            if (response.status === 200) {
+                message.textContent = data.msg
+                thisEvent = new Event('startDisplay')
+                document.dispatchEvent(thisEvent)
+
+            } else { // might happen if the list has been updated since last display
+                message.textContent = 'The jobs entry was not found'
+                thisEvent = new Event('startDisplay')
+                document.dispatchEvent(thisEvent)
+            }
+        } catch (err) {
+            message.textContent = 'A communications error has occurred.'
+        }
+        suspendInput = false
+    } 
+})
+
+})
```

Comment:
These two closing braces at the same indentation level imply that there is an indentation issue somewhere in here. Use a tool like prettier to format your code! At this point you know first hand that keeping it perfect by hand is tedious.

Reply:
I have formatted the code